.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,9 +7,7 @@ Pull requests into cqm-models require the following. Submitter and reviewer shou
 - [ ] Code diff has been done and been reviewed
 - [ ] Tests are included and test edge cases
 - [ ] Tests have been run locally and pass
-- [ ] If there were any JavaScript changes, this PR has updated the `dist` directory using `npm run dist`
```

Comment:
Is there a reason why these were removed from the PR template? Are these no longer relevant/do we want to do this in the same PR as other changes?

Reply:
The reasoning was that they are checked automatically by CI/Travis so they don't need to be checkboxes any longer.